src/lib/menu/menu.scss

```diff
@@ -7,11 +7,13 @@
 @import '../core/a11y/a11y';
 
 $mat-menu-vertical-padding: 8px !default;
+$mat-menu-border-radius: 2px !default;
 
 .mat-menu-panel {
-  @include mat-menu-base();
+  @include mat-menu-base(2);
```

Comment:
Can you add a comment clarifying where this elevation number comes from? I couldn't find it in the spec.